<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LabelGame">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Magnets Game</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelUserId">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Enter Your User ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserIdEntry">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="LeftHanded">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Handed</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RightHanded">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Handed</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="FillInfo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveInfo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Info</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelDominantHand">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dominant Hand:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelStrokedArm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stroked Arm:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="StrokedLeftArm">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Arm</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="StrokedRightArm">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Arm</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="StrokedBothArm">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Both Arms</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LabelGameTime">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Elapsed Game Time: </string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelMirroringStrength">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mirroring Strength: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="MirroringStrength">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SetMirroringStrength">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EnableMirroring">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Mirroring</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelGloveIntensity">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Glove Intensity: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="GloveIntensitySlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SplitHCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="LabelGloveIntensityValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelGloveDeadzone">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Glove Deadzone:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="GloveDeadzone">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetGloveDeadzone">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelMirroringStrengthLimit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(No more than 3000)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawShape1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawShape2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Arrow 
And Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawShape3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Triangle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawShape4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Square</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelShapeToDraw">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose a shape to draw</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearCanvas">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Canvas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mirrorCanvasToggle">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mirror Canvas Toggle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="feedbackCanvasToggle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>feedbackCanvasToggle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="canvasToggle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>canvasToggle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
